.travis.yml

```diff
@@ -17,22 +17,40 @@ env:
   matrix:
     - DRIVER="xdebug" DEPENDENCIES="high"
     - DRIVER="phpdbg" DEPENDENCIES="high"
+    - DRIVER="pcov"   DEPENDENCIES="high"
     - DRIVER="xdebug" DEPENDENCIES="low"
     - DRIVER="phpdbg" DEPENDENCIES="low"
+    - DRIVER="pcov"   DEPENDENCIES="low"
   global:
     - DEFAULT_COMPOSER_FLAGS="--no-interaction --no-ansi --no-progress --no-suggest"
 
 before_install:
+  - composer config github-oauth.github.com ${GH_TOKEN}
```

Comment:
Is this still needed?

Reply:
That one I've removed and the builds are passing now, was only necessary when composer was downloading a million composer.json versions for an earlier version of the patch ...